File: Bossinfo.Models.Hospital/HospitalContext.ts

 //改NameSpace就好
 
    import { BossApiModel, Key } from '@boss/decorator/BossApiModel';
    import { I18nNamespace } from '@boss/decorator/i18n/i18nNs';
    @BossApiModel('api/HospitalContext')
    @I18nNamespace('Model.EDI.HospitalContext')
    export class HospitalContext {
        
        
        public his: his1[];
        
        public hospital: hospital[];
        
        public orderregresult: orderregresult[];
        
        public orderregresultdetail: orderregresultdetail[];
    }
    
    /*
    "HospitalContext": {
        
        "his": "his",
        "hospital": "hospital",
        "orderregresult": "orderregresult",
        "orderregresultdetail": "orderregresultdetail"
    }
    */
    

File: Bossinfo.Models.Hospital/HospitalContext.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Data.Entity;

namespace Bossinfo.Models.Hospital
{
    public class HospitalContext : DbContext
    {
        public DbSet<his1> his { get; set; }
        public DbSet<hospital> hospital { get; set; }
        public DbSet<orderregresult> orderregresult { get; set; }
        public DbSet<orderregresultdetail> orderregresultdetail { get; set; }
        public HospitalContext() : base ("name=HospitalContext")
        {
        }

        protected override void OnModelCreating(DbModelBuilder modelBuilder)
        {
            modelBuilder.HasDefaultSchema("hospital");

        }
    }
}
